### PR TITLE
Fixes #23

### LINKED ISSUE #23: Ignore package-lock.json
Since we already have `yarn.lock`. Ignore `package-lock.json` and avoid generating this using a project level .npmrc.

### PR DESCRIPTION
**Pull request by amit-mb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "build": "./build",
     "watch": "webpack --progress --colors --watch"
   },
+  "lock": false,
```
Comment (revathskumar) [inline package.json]: @amit-mb Can you point me to the documentation for this?

**Comment by revathskumar** [comment]
@amit-mb Can have better commit message saying `ignore package-lock.json` or something like that. 😉

File: package.json (hunk 1 above)
Comment (amit-mb) [reply package.json]: Sure, https://docs.npmjs.com/misc/config#package-lock

Comment (amit-mb) [reply package.json]: Actually referred this, https://github.com/npm/npm/issues/16867#issuecomment-324554854

Comment (revathskumar) [reply package.json]: Thats is just a proposal by someone. I don't think npm have any plan for this to implement.

**Comment by emilsoman** [comment]
Yeah `Fixes #...` should go into the description. It works the same way of closing the issue when merged.

**Pull request merged**